gradle.properties

```diff
@@ -0,0 +1,64 @@
+modName = Logistics Pipes
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = LogisticsPipes
+
+modGroup = logisticspipes
+
+# WHY is there no version field?
+# The build script relies on git to provide a version via tags. It is super easy and will enable you to always know the
+# code base or your binary. Check out this tutorial: https://blog.mattclemente.com/2017/10/13/versioning-with-git-tags/
+
+# Will update your build.gradle automatically whenever an update is available
+autoUpdateBuildScript = false
+
+minecraftVersion = 1.7.10
+forgeVersion = 10.13.4.1614
+
+# Select a username for testing your mod with breakpoints. You may leave this empty for a random user name each time you
+# restart Minecraft in development. Choose this dependent on your mod:
+# Do you need consistent player progressing (for example Thaumcraft)? -> Select a name
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = Developer
+
+# Define a source file of your project with:
+# public static final String VERSION = "GRADLETOKEN_VERSION";
+# The string's content will be replaced with your mods version when compiled. You should use this to specify your mod's
+# version in @Mod([...], version = VERSION, [...])
+# Leave these properties empty to skip individual token replacements
+replaceGradleTokenInFile = Tags.java
```

Comment:
Shouldn't this be [LogisticsPipes.java](https://github.com/GTNewHorizons/LogisticsPipes/blob/9c72709d1bc71732594c03bcc655e74eb46c39d0/src/main/java/logisticspipes/LogisticsPipes.java#L163)?